src/Directives.jl

```diff
@@ -21,7 +21,7 @@ parsebracket(:: directive"", text) = ...
 """
 macro directive_str(text) :(Directive{$(quot(symbol(text)))}) end
 
-const DIRECTIVE_MARKER = r"^(\w+):\s*(?s)(.*)"
+const DIRECTIVE_MARKER = r"^(\w+)(:|\s*\n)((?s).*)$"
```

Comment:
Note: That if directives were to take (optional) arguments they could be grabbed from the second group:

    (:.*|\s*\n)